app/lib/dartdoc/dartdoc_runner.dart

```diff
@@ -381,4 +420,19 @@ class DartdocJobProcessor extends JobProcessor {
     await tmpTar.rename(p.join(outputDir, _archiveFilePath));
     _appendLog(logFileOutput, pr);
   }
+
+  Future<PubDartdocData> _loadPubDartdocData(String outputDir) async {
+    final file = new File(p.join(outputDir, _pubDataFileName));
+    if (!file.existsSync()) {
+      return null;
+    }
+    try {
+      final content = await file.readAsString();
+      return new PubDartdocData.fromJson(
+          convert.json.decode(content) as Map<String, dynamic>);
+    } catch (e, st) {
+      _logger.warning('Unable to parse $_pubDataFileName.', e, st);
```

Comment:
It does not, as it will be deleted after the processing is done. The logger's name will contain the package name and version, and we should be able to replicate the issue.